src/scenes/Header/ProfileMenu/ProfileMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import { makeStyles, Typography } from '@material-ui/core';
+import { FC } from 'react';
+import * as React from 'react';
+import { Link } from '../../../components/Routing';
+
+const useStyles = makeStyles(({ typography }) => ({
+  profileMenu: {
+    width: '230px',
+    padding: '16px',
+    position: 'absolute',
+    top: '70px',
+    right: '100px',
+    backgroundColor: 'white',
+    boxShadow: '0px 9px 12px rgba(209, 218, 223, 0.43)',
+    borderRadius: '8px',
+  },
+  menuHeading: {
+    fontWeight: typography.fontWeightMedium,
+    padding: '8px 0',
+    fontSize: 16,
+  },
+  signOut: {
+    padding: '14px 0 6px 0',
+    borderTop: '1px solid #E5E5E5',
+    display: 'block',
+  },
+}));
+
+export const ProfileMenu: FC = () => {
```

Comment:
Check out the `Menu` in `Sidebar`. Let's try to adapt that here. I'm also fine with this being more "basic" than the designs for now.